pennylane/ops/qubit/parametric_ops.py

```diff
@@ -837,18 +857,18 @@ def simplify(self):
         Hadamard(wires=[0])
 
         """
-        p0, p1, p2 = np.mod(self.data, 2 * np.pi)
+        p0, p1, p2 = qml.math.mod(self.data, 2 * np.pi)
 
         if np.allclose(p0, np.pi / 2) and np.allclose(np.mod(self.data[2], -2 * np.pi), -np.pi / 2):
```

Comment:
Looks like this needs to use `qml.math.mod` as well.

Reply:
Changed to `__mod__`